build/Dockerfile

```diff
@@ -1,26 +1,46 @@
-FROM centos:centos8
-
+FROM registry.access.redhat.com/ubi8/ubi:8.1
 # Use this build arg to set any default test script arguments
-ARG RUN_SCRIPT_ARGS=
 ENV RUN_SCRIPT_ARGS=${RUN_SCRIPT_ARGS}
-
+ENV ROBOT_EXTRA_ARGS=${ROBOT_EXTRA_ARGS}
 ENV HOME /tmp
 
 RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
-    dnf install -y git python3 unzip chromium chromedriver &&\
+    dnf install -y git python3 unzip chromium chromedriver redhat-lsb-core &&\
     dnf clean all
 
+## Install yq in the image
+RUN curl -L https://github.com/mikefarah/yq/releases/download/v4.16.2/yq_linux_amd64 -o /usr/bin/yq &&\
+    chmod +x /usr/bin/yq
+
+## Install oc in the container
+COPY build/oc /usr/local/bin/oc
+# COPY oc /usr/local/bin/oc
+RUN chmod 755 /usr/local/bin/oc &&\
+         oc version --client &&\
```

Comment:
Is this line just to show the client version when building the container?

Reply:
yes, it could be useful as log of the build, but if you think it's irrelevant we could remove, np